### PR TITLE
feat(i18n): add translations for tag and category widgets for `ru` and `uk`

### PR DESCRIPTION
**Pull request by BaDos**
Hi!

Please merge this PR.
I have aded translations for tag and category widgets.

Thanks 😉

### CODE REVIEW

File: .gitignore
Comment (CaiJimmy) [inline .gitignore:1]: Could you exclude this change from the pull request please?

**Review by CaiJimmy — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: .gitignore
Comment (BaDos) [reply .gitignore:1]: Removed

Comment (CaiJimmy) [reply .gitignore:1]: Thanks!

**Pull request merged**